### PR TITLE
[mono-move] Centralize V1 error descriptor mapping and check it against V1 in differential tests

### PR DESCRIPTION
**Pull request by vineethk**
## Description

Introduces `mono_move_output::v1_error`, a centralized module that maps MonoMove (V2) typed errors to the status code, sub-status, and message that V1 (`move-vm-runtime`) would have produced for the same fault. This replaces several scattered, per-consumer error classification functions with a single shared mapping.

The `V1ErrorDescriptor` type carries a `StatusCode`, a `V1SubStatus` (which distinguishes a known sub-status, a known absence of one, and an unmodelled one), and a `V1Message` (which distinguishes byte-identical reproductions of V1's text, known absences, and divergent MonoMove-only text). Consumers can use the provenance fields to decide whether a given field is comparable against V1 output.

We added `CHECK-ERROR-PARITY` directive to the differential test harness. It requires both VMs to fail and asserts that MonoMove's failure, mapped into V1 terms, matches the status code, sub-status, and message V1 actually reported. The expected value comes from V1 at run time rather than a literal in the test, so it cannot go stale. This is the only directive that reads the mapped description instead of MonoMove's native error text, so it is what exercises the mapping; it fails if the error has no V1 equivalent, or if the mapping cannot reproduce V1's message or sub-status. Supporting this, `RunResult::Error` now carries the typed `VMInternalError` rather than a rendered string, and load failures attach the typed error as an `anyhow` source so it survives downcast.

`RuntimeError` is also refined: `DivisionByZeroOrOverflow` is split into `DivisionByZero` and `DivisionOverflow` so signed `MIN / -1` can be distinguished from a zero divisor; `CastOutOfRange` now carries the operand value so the error message can embed it; and `InvalidAbortMessage` now carries the `Utf8Error` cause. The `IntValue` type is introduced to hold a widened integer operand in a pointer-sized enum.

The `aptos-transaction-executor` VM status conversion is updated to use `v1_error::describe` instead of its own downcast chain. The unit-test harness likewise classifies runtime and loader failures by `ExecutionErrorKind` rather than per-variant, with `RuntimeError::Unsupported` routed to `TestResult::Unsupported` so a feature MonoMove lacks reports the test as unrunnable rather than failed.

`describe` deliberately maps only three error types (`RuntimeError`, `LoaderError`, `GasExhaustedError`). Every other error type reaching execution reports an invariant violation raised after the loader has run V1's verification, so V1 would have executed the same input successfully; giving those a V1 status would disguise a MonoMove bug as a V1-shaped failure and suppress the mismatch that should catch it.

## How Has This Been Tested?

- Unit tests in `v1_error.rs` cover arithmetic messages, cast messages, vector sub-statuses, message-less errors, abort-message errors, loader errors, V2-only faults, ambiguous deserialization errors, divergent message marking, and the top-level `describe` downcast dispatch.
- `cast_parity.rs` is a new integration test that calls V1's real `Value::cast_*` implementations for every type pair and boundary value and asserts that MonoMove's descriptor matches V1's rendered output. The test guards against vacuous coverage by asserting more than 500 failing casts are compared.
- Existing differential test cases for casts and enum variant mismatches are updated to assert V2-specific substrings now that the messages are more precise.
- `CHECK-ERROR-PARITY` is applied to 8 out-of-range cast cases and 2 out-of-bounds vector cases. Verified non-vacuous by mutation: flipping the cast status to `TYPE_RESOLUTION_FAILURE` and the vector sub-status to `POP_EMPTY_VEC` fails exactly these checks, while the other 221 differential tests still pass — the pre-existing `CHECK-V1/V2-SUBSTR` directives catch neither mutation.

---

> [!NOTE]
> **Medium Risk**
> Changes how committed transaction VM statuses and error messages are produced for MonoMove execution paths; incorrect mapping could misreport failures in simulation/logs while differential tests mitigate regression risk.
> 
> **Overview**
> Adds **`mono_move_output::v1_error`**, a shared mapping from MonoMove’s typed errors (`RuntimeError`, `LoaderError`, `GasExhaustedError`) to V1-style status, sub-status, and message, with provenance (`V1Message`, `V1SubStatus`) so comparisons know when parity is expected.
> 
> **`aptos-transaction-executor`** drops its local runtime/loader → `VMStatus` tables and routes post-prologue failures through `v1_error::describe`, including fallbacks for unmapped errors and V2-only faults.
> 
> **Runtime** refines errors for V1-aligned reporting: split **`DivisionByZero`** vs **`DivisionOverflow`**, **`CastOutOfRange`** carries a **`ReportedIntValue`**, and **`InvalidAbortMessage`** carries the UTF-8 cause; the interpreter populates these at fault sites.
> 
> **Differential tests** gain **`// CHECK-ERROR-PARITY`** (V2 failures mapped to V1 terms vs V1’s actual VM error), preserve typed **`VMInternalError`** through the runner, and apply the directive across arithmetic, casts, globals, and vectors; enum mismatch cases stay on separate V1/V2 substring checks where messages are not comparable. **`cast_parity.rs`** and **`v1_error`** unit tests lock cast and message behavior to V1.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 1f0f1b227228f4cbc45bdc03f9a3fb84b45ba7af. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Comment by vineethk** [comment]
* **#20599** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20599?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20568** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20568?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20559** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20559?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20557** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20557?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20553** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20553?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20538** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20538?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20526** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20526?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20504** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20504?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20475** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20475?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20434** <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20434?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/aptos-labs/aptos-core/20434?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_unpack_mismatch.move [hunk 1]
```diff
@@ -14,7 +14,8 @@ module 0x42::enum_unpack_mismatch {
 }
 
 // RUN: execute 0x42::enum_unpack_mismatch::unpack_as_b --args 0
-// CHECK-SUBSTR: STRUCT_VARIANT_MISMATCH
+// CHECK-V1-SUBSTR: STRUCT_VARIANT_MISMATCH
+// CHECK-V2-SUBSTR: enum variant mismatch
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_unpack_mismatch.move:18]: CHECK-ERROR-PARITY is missing?

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/generics/generic_enum_abort_parity.move [hunk 2]
```diff
@@ -22,4 +22,5 @@ module 0x42::generic_enum_abort_parity {
 // CHECK: results: 7
 
 // RUN: execute 0x42::generic_enum_abort_parity::unpack_as_left --args 1
-// CHECK-SUBSTR: STRUCT_VARIANT_MISMATCH
+// CHECK-V1-SUBSTR: STRUCT_VARIANT_MISMATCH
+// CHECK-V2-SUBSTR: enum variant mismatch
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/tests/test_cases/differential/generics/generic_enum_abort_parity.move:26]: CHECK-ERROR-PARITY is missing?

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_field_divergent_write.move [hunk 3]
```diff
@@ -68,10 +68,12 @@ module 0x43::variant_field_divergent_write {
 // RUN: execute 0x43::variant_field_divergent_write::run_read_y --args 1
 // CHECK: results: 7
 // RUN: execute 0x43::variant_field_divergent_write::run_read_y --args 0
-// CHECK-SUBSTR: STRUCT_VARIANT_MISMATCH
+// CHECK-V1-SUBSTR: STRUCT_VARIANT_MISMATCH
+// CHECK-V2-SUBSTR: enum variant mismatch
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_field_divergent_write.move:72]: `CHECK-ERROR-PARITY` is missing here and in other files? I imagine we always want to use `CHECK-ERROR-PARITY` for any failure, substring checks only specify that some particular failures occurred.

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/casts/int_cast_widths.masm [hunk 4]
```diff
@@ -222,17 +222,17 @@ fun u8_to_i256(a: u8): i256
 // CHECK: results: 127
 // RUN: execute 0x1::cast_widths::i8_to_u8 --args -1
 // CHECK-V1-SUBSTR: ARITHMETIC_ERROR
-// CHECK-V2-SUBSTR: value out of range
+// CHECK-V2-SUBSTR: Cast.i8->u8: value -1 out of range for u8
 // RUN: execute 0x1::cast_widths::i8_to_u64 --args -1
 // CHECK-V1-SUBSTR: ARITHMETIC_ERROR
-// CHECK-V2-SUBSTR: value out of range
+// CHECK-V2-SUBSTR: Cast.i8->u64: value -1 out of range for u64
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/tests/test_cases/differential/casts/int_cast_widths.masm:239]: Add `CHECK-ERROR-PARITY`?

File: third_party/move/mono-move/testsuite/src/runner.rs [hunk 5]
```diff
@@ -60,8 +61,65 @@ use move_vm_types::{gas::UnmeteredGasMeter, loaded_data::runtime_types::Type};
...
 /// Execution output from a VM as a normalized display string.
-struct Output {
-    display: String,
+pub(crate) struct Output {
+    pub(crate) display: String,
+    pub(crate) v1_error: V1Error,
+}
+
+/// A step's failure stated in V1 terms, for `CHECK-ERROR-PARITY`.
+pub(crate) enum V1Error {
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/src/runner.rs]: Can we call this `CanonicalError` or `NormalizedError`? It is not V1 (this is implementation detail) but rather some normalized representation we can compare between VMs?

File: third_party/move/mono-move/testsuite/src/runner.rs [hunk 6]
```diff
@@ -60,8 +61,65 @@ use move_vm_types::{gas::UnmeteredGasMeter, loaded_data::runtime_types::Type};
 use std::{path::Path, sync::OnceLock};
 
 /// Execution output from a VM as a normalized display string.
-struct Output {
-    display: String,
+pub(crate) struct Output {
+    pub(crate) display: String,
+    pub(crate) v1_error: V1Error,
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/src/runner.rs]: can be just `error`, or since you have `NoFailure` variant I would say we should call this `status: NormalizedStatus` or so?

File: third_party/move/mono-move/testsuite/src/runner.rs [hunk 7]
```diff
@@ -60,8 +61,65 @@ use move_vm_types::{gas::UnmeteredGasMeter, loaded_data::runtime_types::Type};
...
+}
+
+/// A step's failure stated in V1 terms, for `CHECK-ERROR-PARITY`.
+pub(crate) enum V1Error {
+    /// The step failed, and the failure can be stated in V1 terms.
+    Comparable(String),
+    /// The step failed, but MonoMove cannot say what V1 reports for it.
+    Unstatable,
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/src/runner.rs]: nit: `Incomparable` to make it the opposite of the other variant?

File: third_party/move/mono-move/testsuite/src/runner.rs [hunk 8]
```diff
@@ -60,8 +61,65 @@ use move_vm_types::{gas::UnmeteredGasMeter, loaded_data::runtime_types::Type};
...
+    /// The step ended in a Move abort, which carries no VM error to state.
+    Aborted,
+    /// The step did not fail.
+    NoFailure,
+}
+
+impl Output {
+    fn new(display: String) -> Self {
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/src/runner.rs]: nit: call this `success`?

File: third_party/move/mono-move/testsuite/src/runner.rs [hunk 9]
```diff
@@ -60,8 +61,65 @@ use move_vm_types::{gas::UnmeteredGasMeter, loaded_data::runtime_types::Type};
...
+fn render_v1_error(status: StatusCode, sub_status: Option<u64>, message: Option<&str>) -> String {
+    format!(
+        "{:?} | sub-status {} | {}",
+        status,
+        sub_status.map_or_else(|| "none".to_string(), |sub| sub.to_string()),
+        message.unwrap_or("no message"),
+    )
+}
+
+/// MonoMove's failure in V1 terms. `Unstatable` when the error has no V1
```
Comment (georgemitenkov) [inline third_party/move/mono-move/testsuite/src/runner.rs]: Here and above: these are just API to normalize errors, can we maybe not refer in comments to internal details like "`Unstatable` when ...". We can just say if V2 error cannot be normalized, it will not be comparable to any other normalized errors in plain English.

File: third_party/move/mono-move/runtime/src/error.rs [hunk 10]
```diff
@@ -16,8 +20,12 @@ pub enum RuntimeError {
     #[error("{op}.{ty}: underflow")]
     ArithmeticUnderflow { op: ArithOp, ty: IntTy },
 
-    #[error("{op}.{ty}: division by zero")]
-    DivisionByZero { op: ArithOp, ty: IntTy },
+    #[error("{op}: division by zero")]
+    DivisionByZero { op: ArithOp },
+
+    /// Signed `MIN / -1`, whose quotient is not representable.
```
Comment (georgemitenkov) [inline third_party/move/mono-move/runtime/src/error.rs]: No need for this comment: `#[error("{op}: MIN divided by -1 overflows")]` below covers all info?

File: third_party/move/mono-move/runtime/src/error.rs [hunk 11]
```diff
@@ -152,6 +167,53 @@ impl IntoExecutionError for RuntimeError {
     }
 }
 
+/// A Move integer value, widened to the signedness-appropriate 256-bit type.
+/// Both variants are needed: no one type spans `I256::MIN ..= U256::MAX`.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum IntValue {
```
Comment (georgemitenkov) [inline third_party/move/mono-move/runtime/src/error.rs]: Let's call this `DebugIntValue`? so that it is not misused in the future? We only need it for errors, maybe also not make it `pub` (but not sure if enum compilation will complain then)?

File: third_party/move/mono-move/output/tests/cast_parity.rs [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+    IntTy::I32,
+    IntTy::I64,
+    IntTy::I128,
+    IntTy::I256,
+];
+
+/// Builds the same operand for both VMs, or `None` if `ty` cannot hold the
+/// candidate.
+fn operand(ty: IntTy, candidate: &IntValue) -> Option<(Value, IntValue)> {
+    let unsigned = match candidate {
```
Comment (georgemitenkov) [inline third_party/move/mono-move/output/tests/cast_parity.rs]: nit: can do let (unsigned, signed) = match candidate { ... }

File: third_party/move/mono-move/output/src/v1_error.rs [hunk 13]
```diff
@@ -0,0 +1,729 @@
...
+//! Describes a MonoMove error the way V1 would have.
+//!
+//! V1 is the MoveVM in `move-vm-runtime` that MonoMove (V2) replaces.
+//!
+//! [`describe`] maps a type-erased [`VMInternalError`] to the
+//! [`V1ErrorDescriptor`] V1 would have produced for the same fault: its status
+//! code, sub-status, and message. Consumers needing V1-shaped errors share this
+//! one mapping instead of keeping their own.
+//!
+//! The mapping is keyed on the error variant. Messages carry their own
```
Comment (georgemitenkov) [inline third_party/move/mono-move/output/src/v1_error.rs]: Can we have some more human text here please? I feel like some things are really obscure and wordy like "Messages carry their own provenance because they are not uniformly reproducible" - what does this mean exactly?

All we need to say I feel is that we have V2 error system and V1, and we try to normalize V2 to V1. I would say that it also might be easier to normalize some things in V1 to V2, like status code to category. So in general, I think having `Normalized...` structs seems more appropriate?

File: third_party/move/mono-move/output/src/v1_error.rs [hunk 14]
```diff
@@ -0,0 +1,729 @@
...
+/// How a descriptor's message relates to V1 output.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum V1Message {
+    /// Reproduces V1's message byte for byte.
+    Verbatim(String),
+    /// V1 attaches no message.
+    Absent,
+    /// MonoMove text used when V1's message cannot be reproduced. Not comparable
+    /// with V1.
+    Divergent(String),
```
Comment (georgemitenkov) [inline third_party/move/mono-move/output/src/v1_error.rs]: Different

File: third_party/move/mono-move/output/src/v1_error.rs [hunk 15]
```diff
@@ -0,0 +1,729 @@
...
+        match self {
+            V1SubStatus::Known(_) | V1SubStatus::Absent => true,
+            V1SubStatus::Unmodelled => false,
+        }
+    }
+}
+
+/// The V1 status code, sub-status, and message for an error.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct V1ErrorDescriptor {
```
Comment (georgemitenkov) [inline third_party/move/mono-move/output/src/v1_error.rs]: let's call this `..Info` - so we keep for descriptors their own VM meaning.

File: third_party/move/mono-move/output/src/v1_error.rs [hunk 16]
```diff
@@ -0,0 +1,729 @@
...
+            )
+        },
+    })
+}
+
+/// Describes a loader fault.
+fn describe_loader_error(err: &LoaderError) -> V1Equivalent {
+    use LoaderError as L;
+    let descriptor = match err {
+        // The address is hex without a `0x` prefix, as V1 renders it.
```
Comment (georgemitenkov) [inline third_party/move/mono-move/output/src/v1_error.rs:291]: Can we just normalize both V1 and V2 to same format? It would be easier than trying to map all to V1 and also more extensible in the future if we want to go other way? Current implementation of normalization/canonicalization can be no-op for V1 and this conversion for V2?

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_unpack_mismatch.move (hunk 1 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_unpack_mismatch.move:18]: `CHECK-ERROR-PARITY` would fail here rather than pass: V1 builds its message from variant names (`expected enum variant A, found B`), and mono carries only the found tag. The substring checks are the strongest available until the fault site carries variant names.

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/generics/generic_enum_abort_parity.move (hunk 2 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/tests/test_cases/differential/generics/generic_enum_abort_parity.move:26]: `CHECK-ERROR-PARITY` would fail here rather than pass: V1 builds its message from variant names (`expected enum variant A, found B`), and mono carries only the found tag. The substring checks are the strongest available until the fault site carries variant names.

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_field_divergent_write.move (hunk 3 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/tests/test_cases/differential/enums/variant_field_divergent_write.move:72]: `CHECK-ERROR-PARITY` would fail here rather than pass: V1 builds its message from variant names (`expected enum variant A, found B`), and mono carries only the found tag. The substring checks are the strongest available until the fault site carries variant names.

File: third_party/move/mono-move/testsuite/tests/test_cases/differential/casts/int_cast_widths.masm (hunk 4 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/tests/test_cases/differential/casts/int_cast_widths.masm:239]: Done, added to all 13 error cases here plus the other differential tests that were missing it (34 sites total).

File: third_party/move/mono-move/testsuite/src/runner.rs (hunk 6 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/src/runner.rs]: Done, instead named `parity: ParityOutcome` (`status` already means `StatusCode` in this file).

File: third_party/move/mono-move/testsuite/src/runner.rs (hunk 7 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/src/runner.rs]: Done, instead named `Unmappable` (says why it can't be compared, since `Aborted`/`NoFailure` also can't).

File: third_party/move/mono-move/testsuite/src/runner.rs (hunk 8 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/src/runner.rs]: Done.

File: third_party/move/mono-move/testsuite/src/runner.rs (hunk 9 above)
Comment (vineethk) [reply third_party/move/mono-move/testsuite/src/runner.rs]: Done.

File: third_party/move/mono-move/runtime/src/error.rs (hunk 10 above)
Comment (vineethk) [reply third_party/move/mono-move/runtime/src/error.rs]: Done.

File: third_party/move/mono-move/runtime/src/error.rs (hunk 11 above)
Comment (vineethk) [reply third_party/move/mono-move/runtime/src/error.rs]: Done, instead named `ReportedIntValue` (these messages reach the committed `VMStatus`, so not debug-only). Can't drop `pub`: it's a field type of a public `RuntimeError` variant used from `mono-move-output`.

File: third_party/move/mono-move/output/tests/cast_parity.rs (hunk 12 above)
Comment (vineethk) [reply third_party/move/mono-move/output/tests/cast_parity.rs]: Done.

File: third_party/move/mono-move/output/src/v1_error.rs (hunk 13 above)
Comment (vineethk) [reply third_party/move/mono-move/output/src/v1_error.rs]: Done.

File: third_party/move/mono-move/output/src/v1_error.rs (hunk 14 above)
Comment (vineethk) [reply third_party/move/mono-move/output/src/v1_error.rs]: Done, instead named `MonoText` (pairs with `Verbatim`/`Absent`; builder is now `with_mono_message`).

File: third_party/move/mono-move/output/src/v1_error.rs (hunk 15 above)
Comment (vineethk) [reply third_party/move/mono-move/output/src/v1_error.rs]: Done, renamed to `V1ErrorInfo`.

File: third_party/move/mono-move/output/src/v1_error.rs (hunk 16 above)
Comment (vineethk) [reply third_party/move/mono-move/output/src/v1_error.rs:291]: The issue is that we need to come up with a normalized format; the V1 format is not the normalized or canonical format. In general, I think we want to allow for V2 to arbitrarily change the errors. Until that happens there is nothing to normalize against, so leaving as is for now.

**Comment by vineethk** [comment]
@georgemitenkov PTAL

File: third_party/move/mono-move/output/src/v1_error.rs [hunk 17]
```diff
@@ -0,0 +1,712 @@
...
+    fn with_message(status: StatusCode, message: impl Into<String>) -> Self {
+        Self {
+            status,
+            sub_status: V1SubStatus::Absent,
+            message: V1Message::Verbatim(message.into()),
+        }
+    }
+
+    /// A fault V1 reports without a message.
+    fn bare(status: StatusCode) -> Self {
```
Comment (georgemitenkov) [inline third_party/move/mono-move/output/src/v1_error.rs]: nit: `with_no_msg_no_sub_status` or `with_no_message`, `bare` is so AI and has no meaning

**Review by georgemitenkov — APPROVED** [review]

File: third_party/move/mono-move/output/src/v1_error.rs (hunk 16 above)
Comment (georgemitenkov) [reply third_party/move/mono-move/output/src/v1_error.rs:291]: Sounds good, can revisit

File: third_party/move/mono-move/output/src/v1_error.rs (hunk 17 above)
Comment (vineethk) [reply third_party/move/mono-move/output/src/v1_error.rs]: done
